### ISSUE TITLE
Got a 404 error when trying to download Debian Jessie

### ISSUE THREAD

**Issue by vico93** [RoliSoft/WSL-Distribution-Switcher#54]
Using Windows 10 b1703 and the latest code from this repo:
```
<redacted>\WSL-Distribution-Switcher>python get-source.py debian:jessie
[*] Fetching official-images info for debian:jessie...
[*] Fetching Dockerfile from repo debuerreotype/docker-debian-artifacts/47859e2e12eadf1ff7f449d0d8b541a68289a40f/jessie...
[!] Failed to fetch Dockerfile from https://raw.githubusercontent.com/debuerreotype/docker-debian-artifacts/47859e2e12eadf1ff7f449d0d8b541a68289a40f/jessie/Dockerfile: HTTP Error 404: Not Found
```

**Comment by bofh0774** [comment]
Same Problem here.
it fails to download any version of Debian.

**Comment by geo-y** [comment]
The "commit" string in dfurl should be replaced by "'dist-amd64'" (or something like that, please have a look at <https://raw.githubusercontent.com/docker-library/official-images/master/library/debian>). The dfurl combination have two lines in "get-source.py".

**Comment by RoliSoft** [comment]
Fixed by db8a8e1.
